client/src/components/Ticketing/ticketingmanager/TicketTypes/tickettypes.tsx

```diff
@@ -36,39 +33,52 @@ const TicketTypes = () => {
       width: 150,
       editable: true,
     },
+    {
+      field: 'concessions',
+      headerName: 'Concessions',
+      width: 150,
+      editable: true,
+    },
     {
       field: 'delete',
       headerName: 'Delete',
       sortable: false,
       width: 100,
       renderCell: (cell) => {
-        return (
-          <Chip label='Delete' color='error' onClick={
-            () => {
-              setConfirmDeletePrompt(true);
-              setCellData(cell);
-            }
-          } />
-        );
+        // Check if the id of the row is 1
+        if (cell.row.id === 1) {
+          // Return a "Default" tag for id 1
+          return <Chip label="Default" color="primary" />;
```

Comment:
At the moment, it kind of looks like the default `Chip` is a clickable button, so it might be better to make it have a `default` color and then possibly a `size='small'` just so it has a visual cue that it there isn't any functionality for that.

Reply:
good idea. made the change.